ecosystem/sep-0024.md

```diff
@@ -154,7 +154,7 @@ Request Parameters:
 
 Name | Type | Description
 -----|------|------------
-`asset_code` | string | The code of the stellar asset the user wants to receive for their deposit with the anchor.
+`asset_code` | string | The code of the stellar asset the user wants to receive for their deposit with the anchor. The value passed must match one of the codes listed in the [/info](#info) response's deposit object.
```

Comment:
❓ This seems to differ from SEP-6, and says it is the asset that the user will receive, not the asset they will deposit. Is that correct?

Reply:
Answered here: https://github.com/stellar/stellar-protocol/pull/837#issuecomment-763161098